core/server/data/schema/schema.js

```diff
@@ -300,5 +300,34 @@ module.exports = {
         created_by: {type: 'string', maxlength: 24, nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    integrations: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        name: {type: 'string', maxlength: 191, nullable: false},
+        slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
+        icon: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
Is this an image path? We use the convention adding `_image` at the end.
`icon_image` sounds a little weird, but maybe there is a better name.
Ignore this if `icon` is not an image path.

Reply:
This is an image path but I didn't realise there was a convention. `icon_image` seems fine, I'll update it 🙂